src/Linters/PackagePrivateLinter.hack

```diff
@@ -0,0 +1,318 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+use type Facebook\HHAST\{
+  ASTLintError,
+  ASTLinter,
+  FunctionToken,
+  INamespaceUseDeclaration,
+  NameToken,
+  NamespaceDeclaration,
+  NamespaceGroupUseDeclaration,
+  NamespaceToken,
+  NamespaceUseClause,
+  NamespaceUseDeclaration,
+  Node,
+  QualifiedName,
+  Script,
+  TypeToken,
+};
```

Comment:
Heh, we should add a linter to pick these all up - as you're in the HHAST namespace, these are all usable without an explicit `use type`

Reply:
Wrote the linter in Slack workspace earlier and forgot to remove these here. Thanks for pointing this!